magprime/configspec.ini

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Yea for now just do it by hand, that's fine.  I have some ideas on how to make puppet write the INI's for each plugin, but we don't need to do it currently.